src/App.tsx

```diff
@@ -1,51 +1,67 @@
+import {
+  NavLink, Routes, Route, Navigate,
+} from 'react-router-dom';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import cn from 'classnames';
+import { TabsPage } from './pages/TabsPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      className="
+        navbar
+        is-light
+        is-fixed-top
+        is-mobile
+        has-shadow
+        has-navbar-fixed-top
+      "
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={({ isActive }) => cn('navbar-item', {
```

Comment:
className is duplicated, so make sense to move `({ isActive }) => cn('navbar-item', { 'is-active': isActive })` to the variable